scripts/histmakers/w_z_gen_dists.py

```diff
@@ -48,6 +51,12 @@
     list(range(0,101)), # this is the same binning as hists from theory corrections
     name = "ptVgen", underflow=False,
 )
+#axis_ptqVgen = hist.axis.Regular(
+#    20, 0., 0.5, name = "ptqVgen", underflow=False
+#)
+axis_ptqVgen = hist.axis.Variable(
+    [round(x, 4) for x in list(np.arange(0, 0.1 + 0.0125, 0.0125))]+[round(x, 4) for x in list(np.arange(0.1+0.025, 0.5 + 0.025, 0.025))], name = "ptqVgen", underflow=False
```

Comment:
it might be simpler and easier to read if you replace "round(x, 4) for x in list(np.arange(0, 0.1 + 0.0125, 0.0125))]" with "[round(j\*0.0125, 4) for j in range(9)]" and similarly for the others, with a form like "[round(start + j\*step, 4) for j in range(nSteps+1)]"